dbms/src/Storages/StorageMergeTree.cpp

```diff
@@ -126,6 +126,13 @@ void StorageMergeTree::drop()
     data.dropAllData();
 }
 
+void StorageMergeTree::truncate()
+{
+    shutdown();
```

Comment:
@alexey-milovidov `shutdown` is synchronous? What would I do if it wasn't ? Because I need to know when the merge task is terminated.

Reply:
It's synchronous.